src/app/Topology/Shared/Entity/EntityDetails.tsx

```diff
@@ -214,24 +229,110 @@ export const TargetDetails: React.FC<{
     ];
   }, [serviceRef]);
 
+  const onToggle = React.useCallback(() => setExpanded((v) => !v), [setExpanded]);
+
   return (
     <DescriptionList {...props} columnModifier={columnModifier}>
-      {_transformedData.map((d) => (
-        <DescriptionListGroup key={d.key}>
-          <DescriptionListTermHelpText>
-            <Popover headerContent={d.helperTitle} bodyContent={d.helperDescription}>
-              <DescriptionListTermHelpTextButton>{d.title}</DescriptionListTermHelpTextButton>
-            </Popover>
-          </DescriptionListTermHelpText>
-          <DescriptionListDescription style={{ userSelect: 'text', cursor: 'text' }}>
-            {d.content}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      ))}
+      {_transformedData.map(mapSection)}
+      <ExpandableSection
+        toggleText={isExpanded ? 'Show less' : 'Show more'}
+        onToggle={onToggle}
+        isExpanded={isExpanded}
+      >
+        <MBeanDetails isExpanded={isExpanded} connectUrl={serviceRef.connectUrl} />
+      </ExpandableSection>
     </DescriptionList>
   );
 };
 
+const MBeanDetails: React.FC<{ isExpanded: boolean; connectUrl: string }> = ({ isExpanded, connectUrl }) => {
+  const context = React.useContext(ServiceContext);
+  const [dayjs, dateTimeFormat] = useDayjs();
+  const addSubscription = useSubscriptions();
+  const [mbeanMetrics, setMbeanMetrics] = React.useState({} as MBeanMetrics);
+
+  React.useEffect(() => {
+    if (isExpanded) {
+      addSubscription(
+        context.api
+          .graphql<MBeanMetricsResponse>(
+            `
+            query MBeanMXMetricsForTarget($connectUrl: String) {
+              targetNodes(filter: { name: $connectUrl }) {
+                mbeanMetrics {
+                  runtime {
+                    startTime
+                    vmVendor
+                    vmVersion
+                  }
+                  os {
+                    version
+                    arch
+                    availableProcessors
+                  }
+                }
+              }
+            }`,
+            { connectUrl }
+          )
+          .pipe(map((resp) => resp.data.targetNodes[0].mbeanMetrics))
```

Comment:
`resp.data` might be null if there is an error. How about adding a `catchError` like this:

```ts
.pipe(
  map((resp) => resp.data.targetNodes[0].mbeanMetrics),
  catchError((_) => of({})) 
)
```

This way, any error will be caught and just set empty.